pkg/policy/rule.go

```diff
@@ -11,16 +11,22 @@ import (
 	"github.com/cilium/proxy/pkg/policy/api/kafka"
 
 	"github.com/cilium/cilium/pkg/identity"
+	ipcachetypes "github.com/cilium/cilium/pkg/ipcache/types"
 	slim_metav1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/meta/v1"
 	"github.com/cilium/cilium/pkg/labels"
 	"github.com/cilium/cilium/pkg/lock"
 	"github.com/cilium/cilium/pkg/option"
 	"github.com/cilium/cilium/pkg/policy/api"
 )
 
+type ruleKey struct {
+	resource ipcachetypes.ResourceID
+	idx      uint
```

Comment:
Index in reference to what? A comment here would be good.

Reply:
sounds good, I'll add a comment in an incoming PR.